account_report_invoice_paid/__manifest__.py

```diff
@@ -0,0 +1,21 @@
+# Copyright (C) 2020-Today: Druidoo (<http://www.druidoo.io/>)
+# @author: Druidoo
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html
+
+{
+    'name': 'Invoice Report Extended',
+    'summary': 'Do changes in account invoice report',
```

Comment:
```suggestion
    'summary': '',
```
Because we prefer the readme